games-strategy/maxr/maxr-0.2.9.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```

Comment:
Please use EAPI 6. It probably doesn't require any changes.

Reply:
done